scripts/qa_crowbarsetup.sh

```diff
@@ -1978,9 +1976,9 @@ function get_novadashboardserver()
 function get_ceph_nodes()
 {
     if [[ -n "$deployceph" ]]; then
-        cephmons=`crowbar ceph proposal show default | $ruby -e "require 'rubygems';require 'json';puts JSON.parse(STDIN.read)['deployment']['ceph']['elements']['ceph-mon']"`
-        cephosds=`crowbar ceph proposal show default | $ruby -e "require 'rubygems';require 'json';puts JSON.parse(STDIN.read)['deployment']['ceph']['elements']['ceph-osd']"`
-        cephradosgws=`crowbar ceph proposal show default | $ruby -e "require 'rubygems';require 'json';puts JSON.parse(STDIN.read)['deployment']['ceph']['elements']['ceph-radosgw']"`
+        cephmons=`crowbar ceph proposal show default | rubyjsonparse "puts j['deployment']['ceph']['elements']['ceph-mon']"`
+        cephosds=`crowbar ceph proposal show default | rubyjsonparse "puts j['deployment']['ceph']['elements']['ceph-osd']"`
+        cephradosgws=`crowbar ceph proposal show default | rubyjsonparse "puts j['deployment']['ceph']['elements']['ceph-radosgw']"`
```

Comment:
Would be good to add another commit on top of this (maybe in a new PR when this one is merged) which does further refactoring here, e.g.:

- [ ] Just call `crowbar ceph proposal show default` once, and save the result for reuse.
- [ ] Eliminate the `rubyjsonparse "puts j['deployment']['ceph']['elements']..."` duplication with another shell function